src/pages/Product/Product.js

```diff
@@ -1,30 +1,58 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import Color from './components/Color/Color';
+import Count from './components/Count/Count';
+import Review from './components/Review/Review';
+import ColorButton from './components/ColorButton/ColorButton';
 import "./Product.scss";
 
 const Product = () => {
-  const price = 300;
-  const totalPrice = price;
+  const [comment, setComment] = useState(true);
+  const [count, setCount] = useState(1);
+  const [color, setColor] = useState('white');
+  const [dataList, setDataList] = useState();
+
+  useEffect(() => {
+    fetch('/data/data.json', {
+      method: 'GET'
+    })
```

Comment:
```suggestion
    fetch('/data/data.json')
```

method가 GET 요청일 때, header나 다른 정보가 없다면 생략해도 좋습니다.

Reply:
한줄로 사용해도 될줄은 몰랐는데 감사합니다!👍